packer_images/sweeper_dead_letter_pipeline.conf

```diff
@@ -0,0 +1,13 @@
+input {
```

Comment:
since this file is identical to the existing one, I think we don't need it for now. You can just copy in the Sweeper image the `dead_letter_pipeline.conf`